common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -503,3 +511,41 @@ def test_discussion_on_team_page(self):
         self.assertTrue(discussion.is_discussion_expanded())
         self.assertEqual(discussion.get_num_displayed_threads(), 1)
         self.assertTrue(discussion.has_thread(thread['id']))
+        assertion = self.assertTrue if should_have_permission else self.assertFalse
+        assertion(discussion.q(css='.post-header-actions').present)
+        assertion(discussion.q(css='.add-response').present)
+        assertion(discussion.q(css='.new-post-btn').present)
+
+    def test_discussion_on_my_team_page(self):
+        """
+        Scenario: Team Page renders a discussion for a team to which I belong.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am a member
+        When the team has a discussion with a thread
+        And I visit the Team page for that team
+        Then I should see a discussion with the correct discussion_id
+        And I should see the thread which I had posted
+        And I should see controls to change the state of the discussion
+        """
+        self.create_membership(self.user_info['username'], self.team['id'])
+        self.verify_teams_discussion_permissions(True)
+
+    def test_discussion_on_other_team_page(self):
+        """
+        Scenario: Team Page renders a team discussion for a team to which I do
+            not belong.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am not a member
+        When the team has a discussion with a thread
+        And I visit the Team page for that team
+        Then I should see a discussion with the correct discussion_id
+        And I should see the thread which had been posted
+        And I should not see the thread which I had posted
+        And I should not see controls to change the state of the discussion
+        """
+        self.verify_teams_discussion_permissions(False)
+
+    @ddt.data('Moderator', 'Community TA', 'Administrator')
```

Comment:
For completeness, I like having all 3 of these. But how much time does it take to run all 3 test cases? One out of the 3 might suffice... I will let you decide.